res/controllers/Traktor-Kontrol-Z1-scripts.js

```diff
@@ -58,12 +54,14 @@ function KontrolZ1Controller() {
                 }
             })
             controller.setCallback('control', 'hid', channel + '_button_fx', function (button) {
-                const ch = button.name.substr(0, 1)
+                var ch = button.name.substr(0, 1)
                 if (button.value === controller.buttonStates.pressed) {
                     if (controller.modifiers.get('mode')) {
                         controller.toggle('[Channel' + ch + ']', 'play')
                     } else {
-                        controller.toggle('[QuickEffectRack1_[Channel' + ch + ']_Effect1]', 'enabled')
+                        var newVal = engine.getParameter('[QuickEffectRack1_[Channel' + ch + ']_Effect1]', 'enabled') ? 0 : 1
+                        engine.setParameter('[EqualizerRack1_[Channel' + ch + ']_Effect1]', 'enabled', newVal)
```

Comment:
Why is this now disabling the EQ?

Reply:
I found it much more useful this way, because I don't really see why you would need a button to disable a single effect. But I was also unsure whether/how to include it.

Reply:
I don't think I've ever seen a button on a DJ controller or mixer to disable the EQ. I don't know what a use case for that would be. I'm sure this button is only intended to toggle the filter on/off.

Reply:
I had a usage for it, I wanted to disable all EQ at once right before the drop because I also cut away another song at that time. If you only need to disable the filter you might as well just turn the knob directly.